File: 2534 - General Exam.py
while True:
    grades = []
    try:
        line = input()
        NQ = line.split()
        N = int(NQ[0])
        Q = int(NQ[1])

        for i in range(N):
            temp = int(input())
            grades.append(temp)
        grades.sort(reverse=True)
        #print(grades)

        for i in range(Q):
            queries = int(input())
            print(grades[queries-1])



    except EOFError:
        break

File: 2534 - General Exam.c
#include <stdio.h>
int main()
{
    int i,j,N,Q,temp;

    while(scanf("%d", &N) >= 0){

            int new_arr[N];
            scanf("%d", &Q);

            for(i=0;i<N;i++)
            {
                scanf("%d", &temp);
                new_arr[i]=temp;
            }

            //sort
            for(i=0;i<N;i++){
                for(j=i;j<N;j++){
                    if(new_arr[i]<new_arr[j])
                    {
                        temp=new_arr[i];
                        new_arr[i]=new_arr[j];
                        new_arr[j]=temp;
                    }
                }
            }


            for(i=0;i<Q;i++)
            {
                scanf("%d", &temp);
                printf("%d\n",new_arr[temp-1]);

            }

    }

    return 0;
}
